fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -1128,6 +1128,10 @@ create_stmt ::=
         RESULT = new CreateTableStmt(ifNotExists, isExternal, name, columns, indexes, engineName, keys, partition,
         distribution, tblProperties, extProperties, tableComment, index);
     :}
+    | KW_CREATE opt_external:isExternal KW_TABLE opt_if_not_exists:ifNotExists table_name:name KW_LIKE table_name:existed_name
```

Comment:
is `opt_external:isExternal` necessary?

Reply:
yes, it is unnecessary, but if I remove it, it will cause shift/reduce conflict, maybe you can help me how to fix it?